### ISSUE TITLE
Buffer Type is not supported

### ISSUE THREAD

**Issue by egandro** [lukeautry/tsoa#38]
```
export interface FooData {
    id: string;
    jsonData: Buffer;
}
```
Error: No matching model found for referenced type Buffer

We should map Buffer to string.

**Comment by lukeautry** [comment]
@egandro Take a look at this PR: https://github.com/lukeautry/tsoa/pull/40

Here is the swagger.json section that gets generated:

```
"/GetTest/HandleBufferType": {
			"get": {
				"operationId": "getBuffer",
				"produces": [
					"application/json"
				],
				"responses": {
					"200": {
						"description": "",
						"examples": {},
						"schema": {
							"type": "string",
							"format": "base64"
						}
					}
				},
				"description": "",
				"parameters": [
					{
						"description": "",
						"in": "query",
						"name": "buffer",
						"required": true,
						"type": "string",
						"format": "base64"
					}
				]
			}
		},
```

Let me know how this lines up with what you were expecting.

**Comment by lukeautry** [comment]
@egandro The PR referenced above has been merged, so this is live in 1.0.13. Let me know how it works out for you.
